docs/DeveloperGuide.md

```diff
@@ -894,9 +918,74 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. F.A.K.E.J.A.R.V.I.S. shows an error message.
+  * 3a1. F.A.K.E.J.A.R.V.I.S. shows an error message.
+
+    Use case resumes at step 2.
+
+
+**Use case: View list of tasks**
+
+**MSS**
+
+1.  User requests to view tasks
+2.  F.A.K.E.J.A.R.V.I.S. shows a list of tasks matching search criteria.
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. No fields provided
+
+  * 1a1. F.A.K.E.J.A.R.V.I.S. shows list of all tasks.
+
+    Use case ends.
+
+* 1b. Multiple fields provided
+
+  * 1b1. F.A.K.E.J.A.R.V.I.S. shows an error message.
+
+    Use case ends.
+
+* 1c. Invalid fields provided
+
+  * 1c1. F.A.K.E.J.A.R.V.I.S. shows an error message.
+
+    Use case ends.
+
+* 2a. The list is empty.
+
+  Use case ends.
```

Comment:
add an indent here for `Use case ends.`

Reply:
This is fine is it not? It ends at 2a, not a deeper hierarchy